### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by ChrisLoiselle**
adding paragraph test text

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
 		</section>
 		<section>
 			<h2>Introduction</h2>
+			<p> this is a test </p>
```
Comment (ChrisLoiselle) [inline index.html:19]: This is my comment.

**Review by ChrisLoiselle — COMMENTED** [review]
I've reviewed my comments.

File: index.html (hunk 1 above)
Comment (maryjom) [reply index.html:19]: Looks like the editing process is working well.

**Comment by maryjom** [comment]
@ChrisLoiselle I'm closing your test pull request. You can delete your branch.

**Pull request closed without merging**